Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.py

```diff
@@ -0,0 +1,3105 @@
+    endpoint_ids = argToList(args.get('endpoint_ids'))
+    incident_id = arg_to_number(args.get('incident_id'))
+    timeout = arg_to_number(args.get('timeout', 600)) or 600
+    file_paths = argToList(args.get('file_path'))
+    all_files_response = []
+    for file_path in file_paths:
+        parameters = {'path': file_path}
+        response = client.run_script('414763381b5bfb7b05796c9fe690df46', endpoint_ids, parameters, timeout, incident_id)
+        reply = response.get('reply')
+        all_files_response.append(CommandResults(
+            readable_output=tableToMarkdown(f'Run Script File Exists on {file_path}', reply),
+            outputs_prefix=f'{INTEGRATION_CONTEXT_BRAND}.ScriptRun',
+            outputs_key_field='action_id',
+            outputs=reply,
+            raw_response=reply,
+        ))
+    return all_files_response
+
+
+def run_script_kill_process_command(client: Client, args: Dict) -> List[CommandResults]:
+    endpoint_ids = argToList(args.get('endpoint_ids'))
+    incident_id = arg_to_number(args.get('incident_id'))
+    timeout = arg_to_number(args.get('timeout', 600)) or 600
+    processes_names = argToList(args.get('process_name'))
+    all_processes_response = []
+    for process_name in processes_names:
+        parameters = {'process_name': process_name}
+        response = client.run_script('fd0a544a99a9421222b4f57a11839481', endpoint_ids, parameters, timeout, incident_id)
+        reply = response.get('reply')
+        all_processes_response.append(CommandResults(
+            readable_output=tableToMarkdown(f'Run Script Kill Process on {process_name}', reply),
+            outputs_prefix=f'{INTEGRATION_CONTEXT_BRAND}.ScriptRun',
+            outputs_key_field='action_id',
+            outputs=reply,
+            raw_response=reply,
+        ))
+
+    return all_processes_response
+
+
+def add_exclusion_command(client: Client, args: Dict) -> CommandResults:
+    name = args.get('name')
+    if not name:
+        raise DemistoException("Didn't get name arg. This arg is required.")
+    indicator = args.get('filterObject')
+    if not indicator:
+        raise DemistoException("Didn't get filterObject arg. This arg is required.")
+    status = args.get('status', "ENABLED")
+    comment = args.get('comment')
+
+    res = client.add_exclusion(name=name,
+                               status=status,
+                               indicator=json.loads(indicator),
+                               comment=comment)
+
+    return CommandResults(
+        readable_output=tableToMarkdown('Add Exclusion', res),
+        outputs={f'{INTEGRATION_CONTEXT_BRAND}.exclusion.rule_id(val.rule_id == obj.rule_id)': res.get("rule_id")},
+        raw_response=res
+    )
+
+
+def delete_exclusion_command(client: Client, args: Dict) -> CommandResults:
+    alert_exclusion_id = arg_to_number(args.get('alert_exclusion_id'))
+    if not alert_exclusion_id:
```

Comment:
maybe this should be config as required in the yml?

Reply:
It is configed, I wrote the error when I tested with demistomock, can remove it

Reply:
Removed